app/controllers/schools/participants_controller.rb

```diff
@@ -8,9 +8,11 @@ class Schools::ParticipantsController < Schools::BaseController
   before_action :set_mentors_added, only: %i[index show]
 
   def index
-    @participant_profiles = policy_scope(
-      @school_cohort.active_ecf_participant_profiles.includes(:user).order("users.full_name"),
-    )
+    @ineligible = ineligible_participants
+    @eligible = eligible_participants
+    @contacted_for_info = contacted_for_info_participants
+    @details_being_checked = details_being_checked_participants
+    @participant_profiles = participant_profiles
```

Comment:
Could we control this with the `eligibility_notifications` feature flag please? When the flag is on, this is correct. When the flag is off, `@eligible` and `@ineligible` should be empty, and `@details_being_checked` should also contain the eligible and ineligible participants